src/language-css/printer-postcss.js

```diff
@@ -538,7 +538,8 @@ function maybeToLowerCase(value) {
     value.includes("@") ||
     value.includes("#") ||
     value.startsWith("%") ||
-    value.startsWith("--")
+    value.startsWith("--") ||
+    (value.includes("(") && value.includes(")"))
```

Comment:
This can match `(` and `)` out of order (`) foo (`) so perhaps only check `value.includes("(")`? Or perhaps it doesn’t matter. Or it can’t happen due to how the parsing works.

Reply:
@lydell we have `maybeToLowerCase(n.prop)`, `value.includes("(")` not enough, it can break css hacks (example - `.selector { (;property: value;); }`) see http://browserhacks.com/. I don't known should be handle css hacks, but i think we don't break code.

Reply:
Good, then we’re good to go. 👍